<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunningRecordClearPage</class>
 <widget class="QWidget" name="RunningRecordClearPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 20px, &quot;SimHei&quot;;	
}

QWidget
{
	background-color: black;
}

QPushButton
{
	font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
}

QPushButton:disabled
{
		border-radius:5px;
	font: 20px, &quot;SimHei&quot;;
    color: grey;
	background-color: rgb(221,221,221);
}</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>400</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>记录容量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ServiceCapacity">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>400</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_TractionCapacity">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>400</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Ap1Capacity">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>400</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_AuxCostCapacity">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>350</y>
     <width>400</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Ap2Capacity">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>400</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>580</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>240</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ServiceDistance">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>240</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>检修运行里程(km)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_TractionCost">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>240</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>牵引能耗
再生能耗(kmh)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Ap1Time">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>240</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TC1空压机运行时间(kmh)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Ap2Time">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>240</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TC2空压机运行时间(kmh)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_AuxCost">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>240</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>辅助能耗(kmh)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tractionDisableInfo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>490</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15px, &quot;SimHei&quot;;
border:0px;
color:red;</string>
   </property>
   <property name="text">
    <string>牵引和制动能量计数禁止清零</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_SIVDisableInfo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>520</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15px, &quot;SimHei&quot;;
border:0px;
color:red;</string>
   </property>
   <property name="text">
    <string>辅助能耗禁止清零</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
